colcon-build/action.yaml

```
@@ -38,6 +38,10 @@ inputs:
     description: Will be exported as MAKEFLAGS environment variable. e.g. "-j 4"
     default: ""
     required: false
+  underlay-workspace:
+    description: Path to the underlay workspace containing setup.sh
+    default: /opt/autoware
```

```suggestion
    default: ""
```

I'd suggest making the default empty, or even Universe and any other image that depends on autoware images will start using this directory, potentially causing complications.